frontend/views/containers/proposals/AddMembers.vue

```diff
@@ -111,71 +99,49 @@ export default {
     }
   },
   methods: {
-    async addInvitee (index) {
-      const searchUser = this.form.invitees[index]
-
-      if (searchUser === this.ourUsername) {
-        return this.setInviteError(index, this.L("You can't invite yourself"))
-      }
-
-      if (this.currentGroupState.profiles[searchUser]) {
-        return this.setInviteError(index,
-          this.L('{searchUser} is already on the group', { searchUser })
-        )
-      }
-
-      try {
-        const contractID = await sbp('namespace/lookup', searchUser)
-        if (!contractID) {
-          this.setInviteError(index, this.L('Non existing user. TODO support general invites'))
-          return
-        }
-        const userState = await sbp('state/latestContractState', contractID)
-        this.ephemeral.invitesToSend[index] = { contractID, ...userState }
-        Vue.set(this.form.eachFeedbackMsg, index, {
-          text: this.L('Ready to be invited!'),
-          class: 'has-text-success'
-        })
+    inviteeUpdate (e, index) {
+      if (e.target.value.length > 0 && !this.ephemeral.isValid) {
         this.ephemeral.isValid = true
-      } catch (err) {
-        console.error(err)
-        this.setInviteError(index, err.message)
       }
-    },
-    setInviteError (index, text) {
-      this.ephemeral.invitesToSend[index] = null
-      Vue.set(this.form.eachFeedbackMsg, index, {
-        text,
-        class: 'error'
-      })
+      if (e.target.value.length === 0 && this.ephemeral.invitesCount === 1 && this.ephemeral.isValid) {
+        this.ephemeral.isValid = false
+      }
     },
     removeInvitee (index) {
       this.ephemeral.invitesCount -= 1
       this.form.invitees.splice(index, 1)
-      this.ephemeral.invitesToSend[index] = null
     },
-    addInviteeSlot () {
+    addInviteeSlot (e) {
       this.ephemeral.invitesCount += 1
+      Vue.nextTick(() => {
+        const inviteeSlots = this.$refs.fieldset.getElementsByTagName('label')
+        const newInviteeSlot = inviteeSlots[inviteeSlots.length - 1]
+        newInviteeSlot && newInviteeSlot.focus()
+      })
+    },
+    resetInvitations () {
+      this.ephemeral.invitesCount = 1
+      this.form.invitees[0] = ''
```

Comment:
Did you try just calling `this.$v.$reset()` instead of these two lines?

Reply:
invitesCount controls the number of inputs to show (1). We are not using vuelidate here because we don't need validations in this form. Besides that, it's a dynamic form, so I decided to keep it simple.